root.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+id="io.github.niccokunzmann.python_dhcp_server"
+
+set -e
+cd "`dirname \"$0\"`"
+
+echo "Uninstall the currently installed application"
+flatpak uninstall -y $id || true
+
+sudo -i flatpak install -y "`pwd`/$id.flatpak"
+sudo -i flatpak run "$id"
```

Comment:
not sure what this, but you never `sudo flatpak run` or even `flaptak install`

Reply:
This must stay here if I want to release this at all. DHCP requires port 67, requires root. If I can share this differently and without root, I would be happy.

See https://discourse.flathub.org/t/network-bind-to-port-below-1024/6736

I will open an issue to resolve this but for now, this is the only way this software can be used.

Reply:
the flathub is NOT suitable it if requires uid 0.

Reply:
Good point! It works for me and it will work for other people when I have documented how to install it.

You can decide to not host it. Then, I will create my own repository. Let me know what you think.